anyway/infographics_utils.py

```diff
@@ -1118,22 +1120,53 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
 
 
 @register
+# TODO: As it currently stands, for SubUrbanWidgets such as this, the is_relevant() method is simply is_sub_urban() which checks for
+# the presence of "road1" and "road_segment_name", which do not match the condition of "street1/2_hebrew" having value, which is required
+#  for matching with the "involved_markers_hebrew" table (see SQLAlchemy below). Maybe we should make InjuredAccidentsWithPedestriansWidget
+# inherit Widget instead of SubUrbanWidget?
 class InjuredAccidentsWithPedestriansWidget(SubUrbanWidget):
     name: str = "injured_accidents_with_pedestrians"
 
+    def validate_parameters(self):
+        # TODO: validate each parameter and display message accordingly
+        return self.request_params.location_info.get('yishuv_name') is not None and \
+            (self.request_params.news_flash_obj.street1_hebrew is not None or   \
+            self.request_params.news_flash_obj.street2_hebrew is not None) and  \
```

Comment:
I think we should remove the validation on street2_hebrew since it's not null only in rare cases for now

Reply:
done